modules/sw360/sw360-client/src/main/java/org/eclipse/sw360/antenna/sw360/client/rest/resource/releases/SW360Release.java

```diff
@@ -38,9 +38,12 @@
     private static final String CHANGESTATUS_KEY = "change_status";
     private static final String COPYRIGHTS_KEY = "copyrights";
     private static final String CLEARINGSTATE_KEY = "clearingState";
+    private static final String HOMEPAGE_KEY = "homepage";
 
     @JsonIgnore
     private boolean isProprietary;
+    @JsonIgnore
+    private String homepageurl;
     private String name;
```

Comment:
The field should now be removed; it is not assigned any more. Also the references from equals() and hashCode() are no longer needed.